### ISSUE TITLE
Error encountered while installing the CellCall package

### ISSUE THREAD

**Issue by jiangh615** [ShellyCoder/cellcall#19]
When I installed CellCall, I encountered the following problems:
When I searched online for a solution, I found that enrichPlot can be installed to resolve it. After I installed cellCall, it was the same error again. I wonder if you have a solution. If so, I would appreciate it.
![QQ截图20211203170947](https://user-images.githubusercontent.com/95456867/144579234-90c85ecf-503a-4e66-a7d7-832b5e683537.png)

**Comment by ShellyCoder** [comment]
the lastest version in enrichplot no ever include the plot_grid function, so you should install enrichplot 1.6.1 in https://www.bioconductor.org/packages/3.10/bioc/html/enrichplot.html.

**Comment by BinghuaLi** [comment]
> the lastest version in enrichplot no ever include the plot_grid function, so you should install enrichplot 1.6.1 in https://www.bioconductor.org/packages/3.10/bioc/html/enrichplot.html.

Hi, I installed enrichplot 1.6.1, but it showed that >= 1.9.3 is required.
Will it possible to update CellCall so that we can also install the lastest enrichplot?



![image](https://user-images.githubusercontent.com/49136259/146563578-270aa689-277e-4d54-82a5-97412ce46078.png)

**Comment by jiangh615** [comment]
Yes, the same problem occurred after I installed enrichplot 1.6.1.

**Comment by jiangh615** [comment]
I have solved the problem. You can install enrichplot1.12.3, and cellcall can be successfully installed.
https://www.bioconductor.org/packages/3.13/bioc/html/enrichplot.html
